package.json

```diff
@@ -66,13 +66,14 @@
     "npm": ">=8.0.0"
   },
   "dependencies": {
-    "@blockly/keyboard-navigation": "^0.1.18",
-    "@blockly/plugin-workspace-search": "^4.0.10",
+    "@blockly/keyboard-navigation": "^0.5.4",
```

Comment:
If we aren't checking in package-lock.json, we need to lock all dependencies to specific versions.